src/main/java/net/dv8tion/jda/core/handle/EventCache.java

```diff
@@ -88,17 +88,11 @@ public synchronized void cache(Type type, long triggerId, long responseTotal, JS
 
     public synchronized void playbackCache(Type type, long triggerId)
     {
-        List<CacheNode> items;
-        try
-        {
-            items = eventCache.get(type).remove(triggerId);
-        }
-        catch (NullPointerException e)
-        {
-            //If we encounter an NPE that means something didn't exist.
+        TLongObjectMap<List<CacheNode>> typeCache = this.eventCache.get(type);
+        if (typeCache == null)
             return;
-        }
 
+        List<CacheNode> items = typeCache.remove(triggerId);
         if (items != null && !items.isEmpty())
         {
```

Comment:
I noticed redundancies in this code. Why do we copy and clear the original if it gets removed from the map?

Reply:
I don't follow. What do you mean?

Reply:
See 19e181c